city-infrastructure-platform/urls.py

```diff
@@ -1,9 +1,17 @@
-from django.contrib import admin
 from django.conf import settings
-from django.urls import path
+from django.urls import include, path
+from django.contrib import admin
+from rest_framework import routers
+from traffic_control import api as traffic_control_api
+
+router = routers.DefaultRouter()
+
+# traffic sign plans
+router.register("trafficsignplans", traffic_control_api.TrafficSignPlanViewSet, basename="traffic sign plan")
```

Comment:
IMO using space in base name would make the url names look a bit weird, like `traffic sign plan-list`